test/integration/preflight/preflight_test.go

```diff
@@ -2,14 +2,47 @@ package preflight
 
 import (
 	"bytes"
+	"flag"
+	"fmt"
 	"os"
 	"os/exec"
 	"path/filepath"
 	"strings"
+	"sync"
 	"syscall"
 	"testing"
 )
 
+var prebuildPlays = []string{
+	"check-yum-update/build.yml",
+}
+
+// TestMain ensure that the necessary build steps are performed
+// (in parallel) prior to running the tests.
+func TestMain(m *testing.M) {
+	flag.Parse()
+	var wg sync.WaitGroup
+	wg.Add(len(prebuildPlays))
+	for _, file := range prebuildPlays {
+		go func() {
+			runPlaybook(file)
+			wg.Done()
+		}()
```

Comment:
passing `file` as an argument here was important, because otherwise we're capturing the loop variable in the closure, leading to gotchas later:

https://play.golang.org/p/Gu2BHb2NLL